docs/1.getting-started/11.testing.md

```diff
@@ -5,36 +5,376 @@ navigation.icon: i-ph-check-circle-duotone
 ---
 
 ::callout
-Test utils are still in development and the API and behavior may change. Currently, it is in preview stage but not yet ready for testing production apps.
-If you are a module author, you can find more specific information in the [Module Author's guide](/docs/guide/going-further/modules#testing)
+If you are a module author, you can find more specific information in the [Module Author's guide](/docs/guide/going-further/modules#testing).
 ::
 
-In Nuxt 3, we have a rewritten version of `@nuxt/test-utils`. We support [Vitest](https://github.com/vitest-dev/vitest) and [Jest](https://jestjs.io) as test runners.
+Nuxt offers first-class support for end-to-end and unit testing of your Nuxt application via `@nuxt/test-utils`, a library of test utilities and configuration that currently powers the [tests we use on Nuxt itself](https://github.com/nuxt/nuxt/tree/main/test) and tests throughout the module ecosystem.
 
 ## Installation
 
+In order to allow you to manage your other testing dependencies, `@nuxt/test-utils` ships with various optional peer dependencies. For example:
+
+- you can choose between `happy-dom` and `jsdom` for a runtime Nuxt environment
+- you can choose between `vitest` and `jest` for end-to-end test runners
+- `playwright-core` is only required if you wish to use the built-in browser testing utilities
+
 ::code-group
 ```bash [yarn]
-yarn add --dev @nuxt/test-utils vitest
+yarn add --dev @nuxt/test-utils vitest @vue/test-utils happy-dom playwright-core
 ```
 ```bash [npm]
-npm i --save-dev @nuxt/test-utils vitest
+npm i --save-dev @nuxt/test-utils vitest @vue/test-utils happy-dom playwright-core
 ```
 ```bash [pnpm]
-pnpm add -D @nuxt/test-utils vitest
+pnpm add -D @nuxt/test-utils vitest @vue/test-utils happy-dom playwright-core
 ```
 ```bash [bun]
-bun add --dev @nuxt/test-utils vitest
+bun add --dev @nuxt/test-utils vitest @vue/test-utils happy-dom playwright-core
+```
+::
+
+## Unit Testing
+
+We currently ship an environment for unit testing code that needs a [Nuxt](https://nuxt.com) runtime environment. It currently _only has support for `vitest`_ (although contribution to add other runtimes would be welcome).
+
+### Setup
+
+1. Add `@nuxt/test-utils/module` to your `nuxt.config` file (optional). It adds a Vitest integration to your Nuxt DevTools which supports running your unit tests in development.
+
+   ```js
+   export default defineNuxtConfig({
+     modules: [
+       '@nuxt/test-utils/module'
+     ]
+   })
+   ```
+
+2. Create a `vitest.config.ts` with the following content:
+
+   ```ts
+   import { defineVitestConfig } from '@nuxt/test-utils/config'
+   
+   export default defineVitestConfig({
+     // any custom Vitest config you require
+   })
+   ```
```

Comment:
Could you also add the script that can be used in `package.json`?
Would `"test:unit": "vitest"` be enough?

Reply:
Yes, it would be 👍